lib/rubycritic.rb

```diff
@@ -1,22 +1,7 @@
-require "rubycritic/configuration"
+require "rubycritic/command_factory"
 
 module Rubycritic
   def self.create(options = {})
```

Comment:
@troessner I agree with that guideline. I say go for it!

Informally I presented @bglusman with a way to use RubyCritic programatically (in his deb_ceiling gem) but I dare to say that compatibility was never a first priority...

Given that you guys are in a refactoring process and that there seems to be some community interest in using RubyCritic programatically... May I suggest you agree on an API so that it can be documented and supported? I would love to see that :smile: